qml/mobile/Main.qml

```diff
@@ -228,14 +228,80 @@ Item {
                     property bool active
                     property bool visibleSetting: true
 
-                    y: parent.height-vkb.height
+                    y: parent.height - vkb.height
                     visible: textrender.activeFocus && visibleSetting
 
                     opacity: vkb.active ? 0.7 : 0.3
                     Behavior on opacity {
                         NumberAnimation { duration: textrender.duration; easing.type: Easing.InOutQuad }
                     }
                 }
+
+                // area to handle keyboard usage
```

Comment:
might make sense to do this inside Keyboard.qml?

Reply:
If the area was size of keyboard, then yes, but now with current mobile UI it's handling full screen.